zulipterminal/model.py

```diff
@@ -88,25 +88,42 @@ def __init__(self, controller: Any) -> None:
 
         self._notified_user_of_notification_failure = False
 
-        self.event_actions: 'OrderedDict[str, Callable[[Event], None]]' = (
+        # Events fetched once at startup
+        self.initial_desired_events: List[str] = [
+            'realm',
+            'presence',
+            'subscription',
+            'message',
+            'update_message_flags',
+            'muted_topics',
+            'realm_user',  # Enables cross_realm_bots
+            'realm_user_groups',
+            'update_display_settings',
+            # zulip_version and zulip_feature_level are always returned in
+            # POST /register from Feature level 3.
+            'zulip_version',
+        ]
+
+        # Events desired while in runtime.
+        self.desired_events_action: 'OrderedDict[str, Callable[[Event], None]]' = (
```

Comment:
I don't think we need to modify this variable at all - and that will keep the main movement of the list into `__init__` a lot clearer.

Reply:
Oh ok, I'll reset this back to the original name.